portal-data-sync/app.js

```diff
@@ -2,15 +2,15 @@
 // BambooHR API custom reports: https://documentation.bamboohr.com/reference/request-custom-report-1
 
 const _ = require('lodash');
-const axios = require('axios');
 const AWS = require('aws-sdk');
-const DatabaseModify = require('./js/databaseModify');
-const Employee = require('./models/employee');
-const EmployeeSensitive = require('./models/employee-sensitive');
-const Logger = require('./js/Logger');
+const axios = require('axios');
+const Employee = require('employee'); // from shared layer
+const EmployeeSensitive = require('employee-sensitive'); // from shared layer
+const DatabaseModify = require('databaseModify'); // from shared layer
+const Logger = require('Logger'); // from shared layer
+
```

Comment:
Same comment here.  Why are you not using the ternary syntax for these `require` statements?